Tasks/SonarQubePostTest/PRCA/ReportProcessor.ps1

```diff
@@ -2,6 +2,80 @@ $ProjectGuidAndFilePathMap = @{}
 $ComponentKeyAndPathMap = @{}
 $ComponentKeyAndRelativePathCache = @{}
 
+
+#region Public
```

Comment:
Agreed. For simplicity I will send a separate PR for this as I need to understand the impact on the tests.